packages/nimble-components/src/theme-provider/tests/tokens.stories.ts

```diff
@@ -185,3 +186,14 @@ export const themeAwareTokens: StoryObj<TokenArgs> = {
 };
 
 themeAwareTokens.storyName = 'Theme-aware Tokens';
+
+// Setting token default values is done as part of the FAST render queue so it needs to be cleared before reading them
+// https://github.com/microsoft/fast/blob/bbf4e532cf9263727ef1bd8afbc30d79d1104c03/packages/web-components/fast-foundation/src/design-token/custom-property-manager.ts#LL154C3-L154C3
+// This uses Storybook's "loaders" feature to await the queue. https://storybook.js.org/docs/html/writing-stories/loaders
+themeAwareTokens.loaders = [
+    async () => ({
+        // Matches syntax in Storybook docs for loaders
+        // eslint-disable-next-line @typescript-eslint/no-confusing-void-expression
+        tokens: await waitForUpdatesAsync()
+    })
```

Comment:
Kinda agree with the linter that this is pretty confusing syntax. Since we don't really need the result of the async task (and I'm not sure what the returned result is) I think we can do:

```suggestion
    async () => {
        await waitForUpdatesAsync();
        return {};
    }
```

If that's an issue could just return some dummy value with a more explicit name we aren't using:

```suggestion
    async () => {
        await waitForUpdatesAsync();
        return {tokensLoaded: true};
    }
```

Reply:
Thanks for the suggestions! The empty object approach seems to work fine.